### ISSUE TITLE
Publish this package to Pypi

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-MAPovertyScale#2]
[Affidavit of Indigency](https://github.com/SuffolkLITLab/docassemble-AffidavitofIndigencyMAVC/blob/579b620628372987faa2e5978f72b510adbf9d79/docassemble/AffidavitofIndigencyMAVC/data/questions/affidavit_body.yml#L4) requires this package, and it'd be nice to have the requirements reflect that. But before we can add to the requirements there, we need to add this package to Pypi.

**Comment by nonprofittechy** [comment]
Done https://pypi.org/project/docassemble.MAPovertyScale/
